File: tuikit/src/main/java/com/tencent/tecentim/messagehelper/MyApplication.java
package com.tencent.tecentim.messagehelper;

import android.app.Application;

public class MyApplication extends Application {

    private static MyApplication instance;

    public static MyApplication instance() {
        return instance;
    }

    @Override
    public void onCreate() {
        super.onCreate();
        instance = this;
    }
}

File: 010kotlin/src/main/java/com/xiaoming/a010kotlin/MyApplication.kt
package com.xiaoming.a010kotlin

import android.app.Application
import android.util.Log
import com.xiaoming.a010kotlin.xiangxuedemo.config.Flag
import com.xiaoming.a010kotlin.xiangxuedemo.database.StudentDatabase

class MyApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        Log.d(Flag.TAG, "MyApplication onCreate run")

        // 初始化
        //StudentDatabase.getDatabase(this)
    }

}